xoa/filter.py

```diff
@@ -927,6 +927,58 @@ def erode_coast(data, until=1, kernel=None, xdim=None, ydim=None):
     return erode_mask(data, until=until, kernel=kernel)
 
 
+def demerliac(da, na_thres=0, dt_tol=0.01):
+    """Apply a dermerliac filter on a data array
+    Note that the data array must have a valid time dimension.
+    When the time step is less than an hour, an interpolation is made on the weights
+    since they are made for hourly time series.
+    Parameters
+    ----------
+    da: xarray.DataArray
+    dt_tol: float
+        Relative tolerance for the time step variability
+    na_thres: float
+        A float between 0 and 1 that defines the allowed level a NaN contamination.
+        See :func:`convolve`.
+    Return
+    ------
+    xarray.DataArray
+    """
+    xoa_warn("Use of demerliac is deprecated,  use tidal_filter instead.")
+    # Get time dimension
+    tdim = xcoords.get_tdim(da, errors="ignore")
```

Comment:
Why not simply calling `tidal_filter` with `filter_name="demerliac"` saying in the docstring that it's a simple "shorcut" ?
It can be seen as a convenient function.